packages/beacon-node/src/network/processor/gossipHandlers.ts

```diff
@@ -615,18 +615,18 @@ function getBatchHandlers(modules: ValidatorFnsModules, options: GossipHandlerOp
         return results;
       }
       // all attestations should have same attestation data as filtered by network processor
-      const {subnet, fork} = gossipHandlerParams[0].topic;
+      const {fork} = gossipHandlerParams[0].topic;
       const validationParams = gossipHandlerParams.map((param) => ({
         attestation: null,
         serializedData: param.gossipData.serializedData,
         attSlot: param.gossipData.msgSlot,
         attDataBase64: param.gossipData.indexed,
+        subnet: param.topic.subnet,
```

Comment:
I think doing this is not enough.

Because `gossipValidatorBatchFn` is still hardcoding all params to have the same topic. See https://github.com/ChainSafe/lodestar/blob/ca29dbb8edad29e1eb5ac2c576059abcb1c82ec7/packages/beacon-node/src/network/processor/gossipValidatorFn.ts#L33